api-doc/api-doc.yaml

```diff
@@ -31,56 +31,51 @@ info:
 servers:
   - url: http://{{baseurl}}project
 tags:
-  - name: projectTemplates
-  - name: projectCategories
-  - name: projectTemplateTasks
-  - name: userProjects
-  - name: reports
-  - name: programs
-  - name: solutions
-  - name: admin
-  - name: cloud-services
+  - name: ProjectTemplates
```

Comment:
@MallanagoudaB this is not title case , keep "Project Templates" across all the services